README.rst

```diff
@@ -48,79 +50,36 @@ through the community's forums such as mailing lists and IRC channels. Anyone
-  You can browse through the reported issues and submit issues of your own.
-
-* **Security-related Issue Reporting and Tracking**: For security-related
-  inquiries or reporting suspected security-related bugs in the Zephyr OS,
-  please send email to vulnerabilities@zephyrproject.org.  We will assess and
-  fix flaws according to our security policy outlined in the Zephyr Project
-  `Security Overview`_.
-
-  Security related issue tracking is done in JIRA.  The location of this JIRA
-  is https://zephyrprojectsec.atlassian.net.
-
-* **Mailing List**: The `Zephyr Development mailing list`_ is perhaps the most convenient
-  way to track developer discussions and to ask your own support questions to
-  the Zephyr project community.  There are also specific `Zephyr mailing list
-  subgroups`_ for announcements, builds, marketing, and Technical
-  Steering Committee notes, for example.
-  You can read through the message archives to follow
-  past posts and discussions, a good thing to do to discover more about the
-  Zephyr project.
-
-* **Chatting**: You can chat online with the Zephyr project developer
-  community and other users in two ways:
-
-  * On `Slack`_: Zephyr has dedicated channels on Slack. To register, use the
-    following `Slack Invite`_.
-
-  * IRC channel #zephyrproject on the freenode.net IRC server. You can use the
-    http://webchat.freenode.net web client or use a client-side application such
-    as pidgin (Note that all discussions have moved to Slack, although we still
-    have many developers still available on the IRC channel).
+Here's a quick summary of resources to help you find your way around:
+
+* **Zephyr Project Website**: https://zephyrproject.org
+* **Releases**: https://zephyrproject.org/developers/#downloads.
+* **Source Code**: https://github.com/zephyrproject-rtos/zephyr is the main
+  repository; https://elixir.bootlin.com/zephyr/latest/source. contains a
+  searchable index
+* **Documentation**: http://docs.zephyrproject.org
+* **Samples and example code**: see `Sample and Demo Code Examples`_
+* **Contributing**: see the `Contribution Guide`_
+* **Wiki**: `Zephyr GitHub wiki`_
+* **Issues**: https://github.com/zephyrproject-rtos/zephyr/issues
+* **Security Issues**: To report security bugs, please send email to
+  vulnerabilities@zephyrproject.org; also see the `Security Overview`_.
+  Security issues are tracked at https://zephyrprojectsec.atlassian.net.
+* **Mailing Lists**: users@lists.zephyrproject.org and
+  devel@lists.zephyrproject.org are the main user and developer mailing lists,
+  respectively. You can join the developer's list and search its archives at
+  `Zephyr Development mailing list`_. The other `Zephyr mailing list
+  subgroups`_ have their own archives and sign-up pages.
+* **Chat**: Zephyr's Slack workspace is https://zephyrproject.slack.com.
+  Use this `Slack Invite`_ to register.
+* **IRC**: ``#zephyrproject`` on Freenode. (Slack is where most activity takes
```

Comment:
I think we've pretty much abandoned the IRC channel now, in favor of slack.

I'd also recommend adding a link to your new "help" doc here.

Reply:
> I think we've pretty much abandoned the IRC channel now, in favor of slack.

OK. That's getting into politics and emotional territory for some, but I'll remove it.

> I'd also recommend adding a link to your new "help" doc here.

I wanted to do that too, but I can't yet because of a chicken-and-egg problem: this is the pull request adding the getting help doc, but I have to refer to it with a real link to the latest docs in this page, which won't exist until this PR is merged and the nightly docs build has a chance to re-run.

OK to do that in a follow-up pull request?

Reply:
You can put IRC pointer under Chat, and I would drop the "but some developers prefer the IRC channel".

Reply:
Done.